taskcluster/ac_taskgraph/release_promotion.py

```diff
@@ -0,0 +1,172 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+
+from mozilla_version.fenix import FenixVersion
```

Comment:
What should this be?

Reply:
Great question! It should be:

```suggestion
 from mozilla_version.maven import MavenVersion 
```

I concede it's not obvious to guess 😅 Here's the context: Fenix is an end-product which used to follow its own version scheme. Now it's almost on track with Gecko versions. Android-components are Android libraries, that is to say: Java libraries packaged by the tool called maven. Maven defines a strict way of defining version numbers which don't match neither Fenix nor Gecko. That's why we should use `MavenVersion`

Reply:
Gotcha, thanks for explaining that detail!